modules/@apostrophecms/schema/lib/addFieldTypes.js

```diff
@@ -205,7 +205,7 @@ module.exports = (self) => {
   self.addFieldType({
     name: 'boolean',
     convert: function (req, field, data, destination) {
-      destination[field.name] = self.apos.launder.boolean(data[field.name], field.def);
```

Comment:
We just want:

```
destination[field.name] = self.apos.launder.boolean(data[field.name]);
```

`false` is a legitimate value and the destination's current value should not be consulted in convert.